ClientLib/src/main/java/com/o3dr/android/client/apis/drone/DroneStateApi.java

```diff
@@ -24,6 +25,21 @@
     public static void arm(Drone drone, boolean arm) {
         Bundle params = new Bundle();
         params.putBoolean(EXTRA_ARM, arm);
+        params.putBoolean(EXTRA_EMERGENCY_DISARM, false);
```

Comment:
@squilter the original method should just delegate the implementation to the new one (i.e: just a call to  `arm(drone, arm, false)` as its method body).
This way, if an implementation update/bugfix needs to be made, only one place needs to be updated.